### DISCUSSION TITLE
佬，这个地方写12：00：00就是到了12点自动开始抢票吗，需不需在11：59点下面的开始抢票呢

### DISCUSSION THREAD

**Discussion by QiyuRafarel** [mikumifa/cppTickerBuy#31] [提问专区]
![image](https://github.com/user-attachments/assets/18a640a6-a0d4-4acb-ade2-3371fc287b04)
佬，这个地方写12：00：00就是到了12点自动开始抢票吗，需不需在11：59点下面的开始抢票呢

**Comment by mikumifa** [comment]
不用，准时设置就好
